### PR TITLE
Data visualization for INTO-CPS-Association/DTaaS#99

### PR DESCRIPTION
**Pull request by yohan-unver**

### CODE REVIEW

File: common/functions/issue99_2.ipynb
Comment (prasadtalasila) [inline common/functions/issue99_2.ipynb:1]: Please remove the result of running the jupyter notebook.
Also please check this in ml-workspace docker container

Comment (prasadtalasila) [reply common/functions/issue99_2.ipynb:1]: Other observations: Please rename this file to cosim-output.ipynb

### Comments on first cell:

3. Add some more commands to show the list of available columns between lines 9 and 11. Have python commands to let user select one column (~~drop down menu~~ python statement in which user changes the column index). Please keep the existing code in lines 11 to 15.
4. Change title in line-16 to "Co-simulation outputs"
5. Change y-axis label on line-23 to "Multi-model connections"
6. Save files to "data/cosim/time-series.png"

### Comments on second cell:

1. Have a separate cell for showing start and end-time limits.
2. Separate cell for selecting start and end-times of categorization plot.
3. Have plot names as "data/cosim/categories.png"

File: common/data/data.csv
Comment (prasadtalasila) [inline common/data/data.csv:1]: please rename and move this file to "cosim/maestro-output.csv"

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

**Comment by yohan-unver** [comment]
I found the script that shows all the columns and let the user select one with a selector (allColumns.ipynb)

**Comment by prasadtalasila** [comment]
merged in PR #8.

**Pull request closed without merging**